cbounter/ht_common.c

```diff
@@ -929,6 +940,21 @@ static PyMethodDef HT_VARIANT(_methods)[] = {
     {"items", (PyCFunction)HT_VARIANT(_HT_iter_KV), METH_NOARGS,
      "Iterates over all key-value pairs."
     },
+    {"iteritems", (PyCFunction)HT_VARIANT(_HT_iter_KV), METH_NOARGS,
+     "Iterates over all key-value pairs."
+    },
+    {"keys", (PyCFunction)HT_VARIANT(_HT_iter_K), METH_NOARGS,
+     "Iterates over all keys."
```

Comment:
This doesn't sound right -- should return a list in Python2. Same for `values()` / `items()`.

Reply:
Yeah, we discussed this at length with @janpom yesterday.
We concluded that iterator is better even despite inconsistency in 2, simply because one such "keys()" call would allocate so much memory, eliminating the whole point of bounter (probably accidentally, from the point of the user).